src/azure-cli/azure/cli/command_modules/containerapp/_utils.py

```diff
@@ -412,12 +412,8 @@ def parse_secret_flags(secret_list):
 
 
 def get_linker_client(cmd):
-    resource = cmd.cli_ctx.cloud.endpoints.active_directory_resource_id
-    profile = Profile(cli_ctx=cmd.cli_ctx)
-    credential, subscription_id, _ = profile.get_login_credentials(
-        subscription_id=get_subscription_id(cmd.cli_ctx), resource=resource)
```

Comment:
`resource` will be dropped by Track 1 SDK authentication removal (https://github.com/Azure/azure-cli/pull/29631).